01week/rockPaperScissors.js

```diff
@@ -9,9 +9,59 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+	// get rid of whitespace and convert to lower case before beginning to check
+	hand1 = hand1.trim().toLowerCase();
+	hand2 = hand2.trim().toLowerCase();
 
-  // Write code here
+	// only accept rock, paper, or scissors
+	if((hand1 === 'rock' || hand1 === 'paper' || hand1 === 'scissors') &&
+		 (hand2 === 'rock' || hand2 === 'paper' || hand2 === 'scissors')){
 
+		// if hand1 is rock
+		// and hand2 is scissors, hand1 wins
+		// and hand2 is paper, hand2 wins
+		// otherwise its a tie
```

Comment:
You don't check for a tie until the last condition, however it's the easiest thing to check for. You should check for a tie first before you check if hand1 is equal to rock.